python/mleap/sklearn/feature_extraction/text.py

```diff
@@ -65,4 +83,74 @@ def serialize_to_bundle(self, transformer, path, model_name):
                   "port": "output"
                    }]
 
-        self.serialize(transformer, path, model_name, attributes, inputs, outputs)
\ No newline at end of file
+        self.serialize(transformer, path, model_name, attributes, inputs, outputs)
+
+
+class TfidfVectorSerializer(MLeapSerializer):
+    pipeline_serializer = PipelineSerializer()
+
+    def __init__(self):
+        super(TfidfVectorSerializer, self).__init__()
+
+    @staticmethod
+    def set_prediction_column(transformer, prediction_column):
+        transformer.prediction_column = prediction_column
+
+    @staticmethod
+    def set_input_features(transformer, input_features):
+        transformer.input_features = input_features
+
+    def serialize_to_bundle(self, transformer, path, model_name):
+        num_features = transformer.idf_.shape[0]
+        vocabulary = [None] * num_features
+
+        for term, termidx in transformer.vocabulary_.iteritems():
+            vocabulary[termidx] = str(term)
+
+        tf_attributes = [
+            ('vocabulary', vocabulary),
+            ('binary', transformer.binary),
+            # no concept of 'min_tf' in scikit's TfidfVectorizer
+            ('min_tf', None)
```

Comment:
Have you tried to use the serialized models to transform some leap frames? 

I think it'll likely fail if it doesn't have min_tf currently, I'll have a look what changes are needed to get the transform to work without min_tf.

Reply:
I'll take a look at transforming leap frames. I'll also do a deep dive into the scikit code and see if there's a way I can infer `min_tf`.